src/XR/features/WebXREyeTracking.ts

```diff
@@ -0,0 +1,151 @@
+import { WebXRFeaturesManager, WebXRFeatureName } from '../webXRFeaturesManager';
+import { WebXRAbstractFeature } from './WebXRAbstractFeature';
+import { WebXRSessionManager } from '../webXRSessionManager';
+import { Observable } from "../../Misc/observable";
+import { Vector3, Quaternion } from "../../Maths/math.vector";
+import { Ray } from "../../Culling/ray";
+import { Nullable } from "../../types";
+
+/**
+ * The WebXR Eye Tracking feature grabs eye data from the device and provides it in an easy-access format.
+ * Currently only enabled for BabylonNative applications, developers intending to use this must enable the Gaze Input capability in their appx package manifest for data to flow.

```

Comment:
>developers intending to use this must enable the Gaze Input capability in their appx package manifest for data to flow. [](http://example.com/codeflow?start=58&length=119)

I don't think this comment should be here, this is super UWP specific and has nothing to do with ios/android/etc